### DISCUSSION TITLE
Is it possible to position the popper diagonally to the target?

### DISCUSSION THREAD

**Discussion by sonikasharma1403** [floating-ui/floating-ui#1671] [Q&A]
Is it possible to position the popper diagonally to the target?

**Comment by atomiks** [comment]
Do have a visualization of what you're trying to achieve?

**Reply by sonikasharma1403** [reply]
<img width="408" alt="image" src="https://user-images.githubusercontent.com/86649832/166644399-155ad7ee-19a7-47a7-8019-5889ab3f912a.png">
I want to achieve something like this

**Reply by atomiks** [reply]
You can use `offset()`:

https://codesandbox.io/s/floating-ui-dom-template-forked-qnhmk7?file=/src/index.js

Here it'll be aligned to the edge of the reference element. It will also flip to the opposite diagonal side appropriately if it doesn't fit.
